functions/Get-DbaSsisEnvironmentVariable.ps1

```diff
@@ -8,7 +8,7 @@ The function communicates directly with SSISDB database, "SQL Server Intergation
 Each parameter (besides SqlInstance and SqlCredential) acts as the filter to only include or exclude particular element
 
 .PARAMETER SqlInstance
-The SQL Server instance.
+The SQL Server instance or set of instances separated by commas
```

Comment:
commas is only when typed in directly.
a user can send multiple instances in a variable or a subexpression
so I wouldn't put it like this